sdk/communication/azure-communication-phonenumbers/samples/siprouting/set_sip_trunks_sample .py

```diff
@@ -0,0 +1,38 @@
+# -------------------------------------------------------------------------
```

Comment:
This file seems to have a space character in the name?

Reply:
Fixed.